blueocean-web/src/main/js/blueocean.js

```diff
@@ -1,3 +1,6 @@
+// Apply polyfills before we do anything.
+require('./polyfills');
```

Comment:
That's needed to run on IE ... the earlier changes were for building (it should build fine now with `-DskipTests`.